jsx/Tabs.js

```diff
@@ -171,7 +171,62 @@ TabPane.propTypes = {
   activeTab: React.PropTypes.string
 };
 
```

Comment:
@HenriRabalais would be great if you could add a comment section to the top of this component for "Usage" like Tabs has, especially to see how the props are used. i'd also really like to (reiterate and) see this as its own component